src/pudl/transform/eia860.py

```diff
@@ -416,6 +416,16 @@ def _core_eia860__generators_energy_storage(
         )
     )
 
+    # capitalize the direct_support generator IDs
+    gen_support_columns = [
+        col
+        for col in storage_df.columns
+        if col.startswith("generator_id_direct_support_")
+    ]
+    storage_df.loc[:, gen_support_columns] = storage_df.loc[
+        :, gen_support_columns
+    ].apply(lambda x: x.str.upper())
+
```

Comment:
oh my... are they uncapitalized in their raw form? this is a yellow flag bc imu we generally do our darndest not to touch the generator IDs bc EIA generally does a good job of not being dinguses with their IDs.

At the very least I think you should add more documentation here about why your doing this (even just copy past what you noted in the pr comment)

Reply:
we decided to manually fix a small handful of these support generators that have the same ID as the `generator_id` except for capitalization.